app_dart/lib/src/service/luci.dart

```diff
@@ -170,6 +170,21 @@ class LuciService {
   ///
   /// Latest builds of each builder will be returned from new to old.
   Future<Iterable<Build>> getBuilds(String repo, bool requireTaskName, List<LuciBuilder> builders) async {
+    final List<Request> searchRequests = prepareSearchRequests(repo, requireTaskName, builders);
+    final BatchRequest batchRequest = BatchRequest(requests: searchRequests);
+    final BatchResponse batchResponse = await buildBucketClient.batch(batchRequest);
```

Comment:
Can we add logging for any failed response?

Reply:
Added.